tenant-base/pom.xml

```diff
@@ -35,7 +35,8 @@
         <vespaversion>${project.version}</vespaversion>
         <target_jdk_version>11</target_jdk_version>
         <compiler_plugin_version>3.8.0</compiler_plugin_version>
-        <surefire_version>2.22.0</surefire_version> <!-- NOTE bjorncs 15.06.2017: Version 2.20 has OoM issues -->
+        <surefire_version>2.22.0</surefire_version>
```

Comment:
Hmm, was already 2.22 here. Right, JUnit 4 tests run just fine, _until the junit-jupiter-engine dependency is included_. With that included, we also need the vintage one ... or so it seems :)